### DISCUSSION TITLE
Replacement for `on: 'auto'` in version 11?

### DISCUSSION THREAD

**Discussion by KeithGillette** [shipshapecode/shepherd#2259] [Q&A]
We've been using Shepherd through the `angular-shepherd` wrapper. Upon upgrade to `angular-shepherd@15.0.0`, which updates to `shepherd@11.0.1`, we found that the previous `on: 'auto'` `PopperPlacement` setting supported by Popper.js has been removed. We found `auto` positioning worked extremely well but the Floating UI placement options result in placements that put the arrow in the wrong place or obscure the element being pointed to with the Shepherd tour overlay. Is `auto` coming back?

**Comment by RobbieTheWagner** [comment]
@KeithGillette thanks for reaching out. Could you please describe the behavior that `auto` gave you and how the current behavior differs?

**Comment by KeithGillette** [comment]
Hi, @RobbieTheWagner. I'm not sure if it's due to a subsequent Floating UI release but in testing again, positioning seems much improved. In my previous testing, the update seemed unusable. Now, the positioning is not as elegant but passable. The arrows still aren't centering in the elements as the were before (shown in the comparison screenshots below) but at least the target is unambiguous and unobscured.

Popper `auto`:
![image](https://user-images.githubusercontent.com/19915029/223514442-4d6c58db-5634-4fa5-9f8c-aaa346f8dd1a.png)

Floating UI `right`:
![image](https://user-images.githubusercontent.com/19915029/223516605-747ef68c-2a02-448a-8bd8-0b6fbc60eb55.png)

Popper `auto`:
![image](https://user-images.githubusercontent.com/19915029/223514815-77020fef-efc4-41b1-b1a4-4a1a762766a6.png)

Floating UI `bottom`:
![image](https://user-images.githubusercontent.com/19915029/223518008-b723b3c3-d9dd-4fd0-8f6d-4a448ed649ac.png)

**Reply by RobbieTheWagner** [reply]
@KeithGillette hmm, it should be better centered than this. Would you be able to create a small reproduction in stackblitz or something similar?
